src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -534,15 +535,57 @@ public override int GetHashCode()
             hash = 31 * hash + CompressAllCollections.GetHashCode();
             return hash;
         }
-        
+
         public DynamicJsonValue ToJson()
         {
             return new DynamicJsonValue
             {
-                [nameof(Collections)] =  new DynamicJsonArray(Collections),
+                [nameof(Collections)] = new DynamicJsonArray(Collections),
                 [nameof(CompressAllCollections)] = CompressAllCollections,
                 [nameof(CompressRevisions)] = CompressRevisions
             };
         }
     }
+
+    public static class DatabaseVersion
+    {
+        public const long BaseVersion = 40_000;
+
+        public const long ThrowDocumentIdTooBigFixVersion = 54_200;
+
+        public static long CurrentVersion = ThrowDocumentIdTooBigFixVersion;
+
+        public class SupportedFeatures
+        {
+            public readonly long DatabaseVersion;
+
+            public SupportedFeatures(long version)
+            {
+                DatabaseVersion = version;
+            }
+
+            public bool BaseLine = true;
+
+            public bool ThrowDocumentIdTooBigFix;
```

Comment:
Is it about a _document id_ or a _revision key_? We're using this boolean to check whether we should `ThrowRevisionKeyTooBig()` - note _revision key_ here